### PR TITLE
Unsound use of str::from_utf8_unchecked in ferris-says

### PR DESCRIPTION
**Pull request by dtolnay**
https://github.com/rust-lang/ferris-says/pull/21

### CODE REVIEW

File: crates/ferris-says/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+```toml
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "ferris-says"
+date = "2024-01-13"
+url = "https://github.com/rust-lang/ferris-says/pull/21"
+informational = "unsound"
+
+[versions]
+patched = [">= 0.3.1"]
```
Comment (dtolnay) [inline crates/ferris-says/RUSTSEC-0000-0000.md:10]: Note that a version 0.3.0 for this crate does not exist. The crate version went from 0.2.1 to 0.3.1 in https://github.com/rust-lang/ferris-says/commit/570d876b7fc1cfcf0f46d74d572f84b8e5287941.

**Comment by alex** [comment]
Out of curiosity, what's the motivation for recording this 2+ year old vuln?

**Comment by dtolnay** [comment]
Until recently (https://github.com/rust-lang/www.rust-lang.org/pull/1848), `ferris-says = "0.2"` was prominently featured on the Rust website, and that version is still unsound. The primary call-to-action button (**"GET STARTED"**) on https://www.rust-lang.org goes to a ferris-says tutorial.

**Comment by alex** [comment]
Got it. Thanks!

**Pull request merged**